test/routes/mail/index.spec.js

```diff
@@ -0,0 +1,70 @@
+import nock from 'nock';
+import assert from 'power-assert';
+import request from 'supertest';
+import app from 'app.js';
+
+const SLACK_POST = process.env.KINDLIZED_SLACK;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).